_rules/line-height-not-important-78fd32.md

```diff
@@ -181,17 +184,17 @@ The [computed][] value of the `line-height` property of this `p` element is **no
 
 #### Failed Example 1
 
-This `p` element has a [computed][] `line-height` equal to the font size, which is below the recommended minimum.
+This `p` element has a [computed][] `line-height` equal to the font size, which is below the required minimum.
```

Comment:
Since the rule now considers the used value and not the computed one, the descriptions should be updated accordingly.
Not sure all examples are in the PR, can you double check that no other description uses "computed" value?
```suggestion
This `p` element has a [used][] `line-height` equal to the font size, which is below the required minimum.
```